packages/zosfiles/__tests__/__system__/cli/download/ds/cli.files.download.ds.system.test.ts

```diff
@@ -179,9 +179,9 @@ describe("Download Data Set", () => {
             expect(response.stderr.toString()).toContain("dataSetName");
         });
 
-        it("should fail due to specified data set name does not existed", async () => {
+        it("should fail due to specified data set name does not exist", async () => {
             const shellScript = path.join(__dirname, "__scripts__", "command", "command_download_data_set.sh");
-            const response = runCliScript(shellScript, TEST_ENVIRONMENT, [dsname + ".dummy"]);
+            const response = runCliScript(shellScript, TEST_ENVIRONMENT, ["dummy." + dsname]);
```

Comment:
Why the order change?
Are we concerned of some other test creating the dataset and not properly cleaning afterwards?

Reply:
Suppose the dsname is "ABC.DEF.GHI". Earlier tests in the suite would have created a file "abc/def/ghi" when downloading the dataset. Then when this test runs, the original behavior was that it would create a folder "abc/def/ghi" to contain the "dummy" file. This was a problem, because it's trying to create a folder with the same name as an existing file. Windows seems to allow this, but on Linux `mkdirSync` fails in this case.

An alternative fix would be to pick a new random dsname at the start of this test to avoid the filename conflict. Would that be a better workaround?

Reply:
I think the changed dsname would be better. It makes the tests consistent.